pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -44,15 +46,25 @@ func Execute(cli podresourcesapi.PodResourcesListerClient, nrtupdaterArgs nrtupd
 		return err
 	}
 
-	resMon, err := NewResourceMonitor(cli, resourcemonitorArgs, rteArgs)
+	cs, err := nrtupdater.GetK8sClient("")
+	if err != nil {
+		return err
+	}
+
+	condIn, err := podreadiness.NewConditionInjector(cs)
```

Comment:
please add a command line flag to turn out this behaviour - which should otherwise be on by default

Reply:
Yes. I remember. In progress